sklearn/manifold/t_sne.py

```diff
@@ -423,18 +423,19 @@ def trustworthiness(X, X_embedded, n_neighbors=5, precomputed=False):
         dist_X = X
     else:
         dist_X = pairwise_distances(X, squared=True)
-    ind_X = np.argsort(dist_X, axis=1)
+    np.fill_diagonal(dist_X, np.inf)
```

Comment:
this deserves a comment

Reply:
This review item has not been addressed.

Reply:
Yep sorry I forgot about this one, I just added a comment